We are building with --Dlibexecdir=lib/$(DEB_TARGET_MULTIARCH) so that the
xapp-sn-watcher binary is correctly placed in an arch-dependent directory.

The other libexec files (applet_constants.py  mate-xapp-status-applet.py)
are installed into arch-deps but arch-all package xapp1-common, and thus
cannot use the $(DEB_TARGET_MULTIARCH) directory.

Since the location of the Python files are not hard-coded in the source but
only in the service and desktop file, we install the .py into /usr/libexec,
and patch the service and aplet files to use the updated path.
---
 status-applets/mate/org.mate.panel.applet.MateXAppStatusAppletFactory.service.in |    2 +-
 status-applets/mate/org.x.MateXAppStatusApplet.mate-panel-applet.in.in           |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/status-applets/mate/org.mate.panel.applet.MateXAppStatusAppletFactory.service.in
+++ b/status-applets/mate/org.mate.panel.applet.MateXAppStatusAppletFactory.service.in
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.mate.panel.applet.MateXAppStatusAppletFactory
-Exec=@libexec@/xapps/mate-xapp-status-applet.py
+Exec=/usr/libexec/xapps/mate-xapp-status-applet.py
--- a/status-applets/mate/org.x.MateXAppStatusApplet.mate-panel-applet.in.in
+++ b/status-applets/mate/org.x.MateXAppStatusApplet.mate-panel-applet.in.in
@@ -1,7 +1,7 @@
 [Applet Factory]
 Id=MateXAppStatusAppletFactory
 InProcess=false
-Location=@libexec@/xapps/mate-xapp-status-applet.py
+Location=/usr/libexec/xapps/mate-xapp-status-applet.py
 Name=XApp Status Applet Factory
 Description=Area where XApp status icons appear
 
